tests/parameter_search/test_p_error_binary_search.py

```diff
@@ -105,9 +84,22 @@ def binary_classification_metric(y_true: numpy.ndarray, y_pred: numpy.ndarray) -
     return (y_pred == y_true).mean()
 
 
+def balanced_accuracy_score_argmax(y_true: numpy.ndarray, y_prob: numpy.ndarray) -> float:
+    """Balanced accuracy score classification metric.
+    Args:
+        y_true (numpy.ndarray): Ground truth
+        y_prob (numpy.ndarray): Model predictions.
+
+    Returns:
+        float: Accuracy.
+    """
+    y_pred = y_prob.argmax(1)
+    return balanced_accuracy_score(y_true, y_pred)
+
+
 @pytest.mark.parametrize("attr, value", [("undefined_attr", 1)])
 @pytest.mark.parametrize(
-    "model_name, quant_type, metric", [("CustomModel", "qat", binary_classification_metric)]
+    "model_name, quant_type, metric", [("CustomModel", "qat", balanced_accuracy_score_argmax)]
```

Comment:
maybe add a comment somewhere on why we now chose this metric instead of the old one !